client/components/RootPage.jsx

```diff
@@ -4,10 +4,16 @@ import { prop } from '../theme';
 const RootPage = styled.div`
   min-height: 100%;
   display: flex;
+  justify-content: start;
```

Comment:
Does this make a difference?  I think it should be ‘flex-start’ and that’s the default value so it probably doesn’t change anything?

Reply:
yes it does when the length of the editor increases

Reply:
sometimes the header goes missing it prevents it from that unexpected behaviour